components/tikv_util/src/security.rs

```diff
@@ -158,18 +162,29 @@ impl SecurityManager {
 
 struct Fetcher {
     cfg: Arc<SecurityConfig>,
+    last_fetch: Arc<Mutex<std::time::Instant>>,
 }
 
 impl ServerCredentialsFetcher for Fetcher {
     fn fetch(&self) -> Result<Option<ServerCredentialsBuilder>, Box<dyn Error>> {
-        let (ca, cert, key) = self.cfg.load_certs()?;
-        let new_cred = ServerCredentialsBuilder::new()
-            .add_cert(cert, key)
-            .root_cert(
-                ca,
-                CertificateRequestType::RequestAndRequireClientCertificateAndVerify,
-            );
-        Ok(Some(new_cred))
+        if let Ok(mut last) = self.last_fetch.try_lock() {
```

Comment:
I am not sure how it solves the problem.

Reply:
Updated this PR detail.